cantabular-import/analysis/report-errors.py

```diff
@@ -35,7 +36,7 @@
 filename = "tmp/all-container-logs.txt"
 
 line_number = 0
-error_found = 0
+race_error_found = 0
```

Comment:
maybe this could be a boolean value? (I think you don't need the count, only a bool to know if there is any data race or not)